<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Event::services::ServiceWidget</class>
 <widget class="QWidget" name="Event::services::ServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblStatus">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/qf/core/images/light-red.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblServiceName">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>EmmaClient</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblServiceMessage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>neco neco</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QToolButton" name="btPlay">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../libqf/libqfgui/images/flat/qf_gui_images_flat.qrc">
          <normaloff>:/qf/gui/images/flat/media-play.svg</normaloff>:/qf/gui/images/flat/media-play.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btShowDetail">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="icon">
         <iconset resource="../../../../../../../libqf/libqfgui/images/flat/qf_gui_images_flat.qrc">
          <normaloff>:/qf/gui/images/flat/settings.svg</normaloff>:/qf/gui/images/flat/settings.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../../../libqf/libqfgui/images/flat/qf_gui_images_flat.qrc"/>
 </resources>
 <connections/>
</ui>
